core/src/Ribasim.jl

```diff
@@ -4,6 +4,7 @@ import IterTools
 import BasicModelInterface as BMI
 
 using Arrow: Arrow, Table
+using Base.Iterators
```

Comment:
I don't think this is needed? `Iterators` is already available, `Iterators.product` works without this.